docs: showcase updated use cases in example workflows (#376)
slackapi/slack-github-action: example-workflows/Technique_2_Slack_API_Method/README.md

# Technique 2: Slack API method

> A bot token or user token or [token of some other kind][tokens] must be used to

Reviewer: wait, so any kind of token is fair game?

Author: Yep! What'ever's passed in here is sent with the API method used 🔍

> A bot token or user token or [token of some other kind][tokens] must be used to

Reviewer: ```suggestion
A [token][tokens], such as a bot or user token, must be used to
```

call one of [the Slack API methods][methods] with this technique.

## Setup

> For details on how to setup this technique in GitHub Actions, read the

Reviewer: ```suggestion
For details on how to set up this technique in GitHub Actions, read the
```

Author: I refuse to believe a rules exists for phrasewords like "setup" and "login". Or the rule is "do the opposite of what seemed right" 😉

[`README.md`][setup].

## Example workflows

1. [**Direct message the author**](#direct-message-the-author): Write to the
   Slack user with a matching email.
2. [**Invite a usergroup to channel**](#invite-a-usergroup-to-channel): Create a
   channel and invite members.

### Direct message the author

> Send a direct message to the user that pushed the most recent commits.

Reviewer: ```suggestion
Send a direct message to the user that pushed the most recent commit.
```

I think? Unless it's like "send it to the users who pushed the last 5 committs"

Author: This is wild, but a `push` might actually contain multiple commits 😳  Super confusing for me!

Reviewer: oh does "recent" consider all commits in that push as coming in at the same time?

Author: Ya! You get both a list of commits **and** the `head_commit` which is, IMO, the more interesting one for this action 🗣️

Reviewer: i feel like this explains why my green squares aren't appearing -- i'm not great about pushing every day, even if im committing. rip
